jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -219,15 +222,107 @@
     suffix:
       - test-unit-integration: # kubernetes-pull-test-unit-integration
           status-context: unit/integration
+          success-comment: ''
+          failure-comment: ''
+          error-comment: ''
+          trigger-phrase: 'unit'
           cmd: |
             export KUBE_VERIFY_GIT_BRANCH="${{ghprbTargetBranch}}"
             ./hack/jenkins/gotest-dockerized.sh
       - verify-all: # kubernetes-pull-verify-all
           status-context: verification
+          success-comment: ''
+          failure-comment: ''
+          error-comment: ''
+          trigger-phrase: 'verify'
           cmd: |
             export KUBE_VERIFY_GIT_BRANCH="${{ghprbTargetBranch}}"
             export KUBE_TEST_SCRIPT="./hack/jenkins/verify-dockerized.sh"
             ./hack/jenkins/gotest-dockerized.sh
+      - build-test-e2e-gce: # kubernetes-pull-build-test-e2e-gce
```

Comment:
I would expect so as the e2e jobs spin up a whole cluster inside the same project. Too much parallelism and we will run out of project quota.